spec.go

```diff
@@ -132,6 +132,7 @@ type Artifacts struct {
 	// ConfigFiles is a list of files that should be marked as config files in the package.
 	ConfigFiles map[string]ArtifactConfig `yaml:"configFiles,omitempty" json:"configFiles,omitempty"`
 	// TODO: other types of artifacts (systtemd units, libexec, etc)
+	Services map[string]ServiceConfig `yaml:"services,omitempty" json:"services,omitempty"`
```

Comment:
How about we call this `systemdUnit` so its explicit as to what it is?
It also means we can add other types of units other than just services.